src/SFA.DAS.ApplyService.InternalApi.UnitTests/RoatpAssessorControllerGetSectorDetailsTests.cs

```diff
@@ -0,0 +1,295 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Apply;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Controllers;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+using SFA.DAS.ApplyService.InternalApi.Services;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.ApplyService.Domain.Apply;
+using SFA.DAS.ApplyService.Domain.Entities;
+using SFA.DAS.ApplyService.Domain.Sectors;
+using SFA.DAS.ApplyService.InternalApi.Types.Assessor;
+
+namespace SFA.DAS.ApplyService.InternalApi.UnitTests
+{
+    [TestFixture]
+    public class RoatpAssessorControllerGetSectorDetailsTests
+    {
+        private readonly Guid _applicationId = Guid.NewGuid();
+        private readonly int _sequenceId = RoatpWorkflowSequenceIds.DeliveringApprenticeshipTraining;
+
+        private readonly int _sectionId =
+            RoatpWorkflowSectionIds.DeliveringApprenticeshipTraining.YourSectorsAndEmployees;
+
+        private readonly string _firstPageId = "7610";
+        private readonly string _secondPage = "7611";
+        private readonly string _thirdPage = "7612";
+        private readonly string _fourthPage = "7613";
```

Comment:
Can these use the constants in RoatpPreambleQuestionBuilder.cs ?

Reply:
They are just made up numbers, and not significant for the test to work - but I know they appear to be genuine real values.  I'll change them to prevent any confusion